### PR TITLE
Add support for jaeger-collector

### PR DESCRIPTION
**Pull request by acsnem**
Add support for jaeger-collector #49

### CODE REVIEW

File: aiozipkin/transport.py [hunk 1]
```diff
@@ -39,7 +39,9 @@ def send(self, record):
 
         # TODO: add status code check
         try:
-            async with self._session.post(self._address, json=data) as resp:
+            headers = {'Content-Type': 'application/json'}
+            async with self._session.post(self._address, json=data,
```
Comment (jettify) [inline aiozipkin/transport.py]: @asvetlov is this expected, I thought client sets `Content-Type`to json if `json` parameter is used.

File: aiozipkin/record.py [hunk 2]
```diff
@@ -42,7 +46,7 @@ def name(self, n):
         return self
 
     def set_tag(self, key, value):
-        self._tags[key] = value
+        self._tags[key] = str(value)
```
Comment (jettify) [inline aiozipkin/record.py]: this line should be updated too in order to make tests green
https://github.com/aio-libs/aiozipkin/blob/73633be49975349ab02681dcfb0f7476a5bfdb48/tests/test_aiohttp_ingegration.py#L64

Comment (acsnem) [reply aiozipkin/record.py]: fixed

**Comment by jettify** [comment]
Will merge tomorrow, lets give a change others to take a look.

As side note, I probably will have to add jaeger docker fixture like we have for zipkin, in order not to break compatibility in future.
https://github.com/aio-libs/aiozipkin/blob/3f5e371a96c90675808cc643f61ed1697fd5d1cb/tests/docker_fixtures.py#L27-L48

File: aiozipkin/record.py (hunk 2 above)
Comment (adriancole) [reply aiozipkin/record.py]: good catch, btw

**Comment by jettify** [comment]
thanks everyone!

**Pull request merged**